pylint/checkers/logging.py

```diff
@@ -393,5 +403,19 @@ def _count_supplied_tokens(args: list[nodes.NodeNG]) -> int:
     return sum(1 for arg in args if not isinstance(arg, nodes.Keyword))
 
 
+def str_formatting_in_f_string(node: nodes.NodeNG) -> bool:
```

Comment:
```suggestion
def str_formatting_in_f_string(node: nodes.JoinedStr) -> bool:
```

Basing this suggestion on these lines:
```python
        elif isinstance(format_arg, nodes.JoinedStr):
            if str_formatting_in_f_string(format_arg):
```